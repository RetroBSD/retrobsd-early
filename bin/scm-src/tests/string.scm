#\a
#\A
#\1
#\9
#\(
#\
#\Space
#\Newline
#\\n
#\\r
#\\t
#\\b
#\\f
#\\33
"qwerty\1\2\3\4\\\"\n\r\t\b\f"
"Hello,
World!"
(string? "abc")
(string? 1)
(make-string 5 #\a)
(string #\s #\c #\h #\e #\m #\e)
