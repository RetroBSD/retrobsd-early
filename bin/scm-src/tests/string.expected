#\a --> #\a
#\A --> #\A
#\1 --> #\1
#\9 --> #\9
#\( --> #\(
#\space --> #\space
#\space --> #\space
#\newline --> #\newline
#\newline --> #\newline
#\\r --> #\\r
#\\t --> #\\t
#\\b --> #\\b
#\\f --> #\\f
#\\033 --> #\\033
"qwerty\001\002\003\004\\\"\n\r\t\b\f" --> "qwerty\001\002\003\004\\\"\n\r\t\b\f"
"Hello,\nWorld!" --> "Hello,\nWorld!"
(string? "abc") --> #t
(string? 1) --> ()
(make-string 5 #\a) --> "aaaaa"
(string #\s #\c #\h #\e #\m #\e) --> "scheme"
